content/events/2022-10-egd/index.md

```diff
@@ -173,22 +173,34 @@ Freiburg can be easily reached by train with the ICE from the North (via Frankfu
 
 ### Lodging
 
-#### City Center
+#### Special discounts for the EGD
+
+There are many options for accommodation in Freiburg. Please **book through this [link](https://tportal.tomas.travel/EuropeanGalaxyDays2022/ukv/?doSearch=1&date_from=03.10.2022&date_to=07.10.2022&number_adult[]=1&searchtext=Freiburg&rate=FIT00020071256464200&reset=1&lang=en)** to get special discounts for the event.
+#### Other options
+##### City Center
 - [Stadthotel Freiburg](https://www.hotel-freiburg.de/de)
 - [Motel One](https://www.motel-one.com/en/hotels/freiburg/hotel-freiburg)
 - [Black Forest Hostel](https://www.blackforest-hostel.de)
 - [Hotel am Rathaus](http://www.am-rathaus.de)
 - [Hotel Barbara](http://www.hotel-barbara.de)
 - [Hotel Bären](https://www.hotel-freiburg-baeren.de/)
 
-#### Main station
+##### Main station
 - [Central](https://www.central-freiburg.de)
 - [Park Hotel Post](https://www.park-hotel-post.de/de)
 - [Intercity Hotel](https://hrewards.com/de/intercityhotel-freiburg)
 
-#### Faculty of engineering
+##### Faculty of engineering (close to the venue)
 - [Hotel Stadt Freiburg](https://www.hotel-stadt-freiburg.de)
 
+# Event rules and regulations
+### CoC
+
+The European Galaxy Days will abide the [Galaxy Project Code of Conduct](https://galaxyproject.org/community/coc/). Please, make sure that you read it before the event.
+### COVID-19 policy
+
+A policy compliant with the national regulations will be in place and announced prior to the event.
```

Comment:
I guess we will be stricter than the national regulations. But I guess this sentence does not hurt for now.